phpBB/memberlist.php

```diff
@@ -1448,6 +1448,20 @@
 				usort($user_list,  'phpbb_sort_last_active');
 			}
 
+			// do we need to display contact fields as such
+			$use_contact_fields = false;
+
+			/**
+			 * Modify the query used to get the users for the team page
```

Comment:
In fact, it allows. Just see what `$use_contact_fields` variable modifies - parameter of `$cp->generate_profile_fields_template_data` call, which modifies SQL query inside that method.